app/models/user_role.rb

```diff
@@ -0,0 +1,7 @@
+# Exposes the `rolify` gem's `users_roles` table ActiveRecord
+class UserRole < ApplicationRecord
```

Comment:
We use this to simplify the algorithm: "select all users that are not suspended", in `Search::Postgres::User`